guides/v2.0/cloud/configure/cloud-vcl-custom-snippets.md

```diff
@@ -87,244 +93,155 @@ The following is an example of a returned JSON for a customer VCL snippet:
 
-1. Edit the bash scripts `staging_snippets.sh` and `production_snippets.sh`.
-2. Change the value for `<SERVICE_ID_VERSION>` with the cloned version number in the line: `export VERSION=<SERVICE_ID_VERSION>`
-3. Save the files.
-4. Run the bash script by entering the file name. The command generates and runs a CURL command for every `.vcl` file in the directory.
+where `<filename.vcl>` is a name of file you prepared on the previous step. Repeat this command for all VCL snippets you want to add.
 
-  * Enter `production_snippets` to run that file and generate VCL snippets to Fastly for your Production environment.
-  * Enter `staging_snippets` to run that file and generate VCL snippets to Fastly for your Staging environment.
-5. As the script runs, VCL snippets should be generating and uploading to Fastly.
+If you get a `500 Internal Server Error` response from Fastly service, check if you are trying to upload a valid JSON file.
 
 ## Validate and activate snippets for a version {#validate}
-When you add the VCL snippet(s) to the version, Fastly creates and assigns it to your service per that version number. Next, you should validate the entered VCL snippets with Fastly. Use the following command to validate all snippets for the version:
+When you add the VCL snippets to the version, Fastly creates and assigns it to your service per that version number. Next, you should validate the entered VCL snippets with Fastly. Use the following command to validate all snippets for the version:
 
-	curl -H "Fastly-Key: {FASTLY_API_TOKEN}" -H 'Content-Type: application/json' -H "Accept: application/json" -X GET https://api.fastly.com/service/{Service ID}/version/{Editable Version #}/validate
+    curl -H "Fastly-Key: ${FASTLY_API_TOKEN}" https://api.fastly.com/service/${FASTLY_SERVICE_ID}/version/${FASTLY_VERSION}/validate
 
 Fastly should return: `"status": "ok"`. If you received an OK, activate the version for that service.
 
 Assuming no errors (if there are errors, fix them before proceeding), the last step is to activate the version with the following command. All VCL snippets associated with the version activate and the previous version snippets deactivate.
 
-	curl -H "Fastly-Key: {FASTLY_API_TOKEN}" -H 'Content-Type: application/json' -H "Accept: application/json" -X PUT https://api.fastly.com/service/{Service ID}/version/{Editable Version #}/activate
+    curl -H "Fastly-Key: ${FASTLY_API_TOKEN}" https://api.fastly.com/service/${FASTLY_SERVICE_ID}/version/${FASTLY_VERSION}/activate -X PUT
 
-If your received errors back from Fastly, track down the errors and update the specific snippet with the following command. Make sure to use the same version number you are working to activate.
+<--If your received errors back from Fastly, track down the errors and update the specific snippet with the following command. Make sure to use the same version number you are working to activate.
```

Comment:
Why the `<--` at the beginning of this line and the `-->` at the end of line 207? Were you trying to comment out  this section using HTML comments (e.g., `<!-- -->`)?

Reply:
My mistake. We can remove this section as well.